Algorithm.CSharp/BasicTemplateOptionsHistoryAlgorithm.cs

```diff
@@ -32,25 +31,19 @@ namespace QuantConnect.Algorithm.CSharp
     /// <meta name="tag" content="history" />
     public class BasicTemplateOptionsHistoryAlgorithm : QCAlgorithm
     {
-        private const string UnderlyingTicker = "GOOG";
-        public readonly Symbol Underlying = QuantConnect.Symbol.Create(UnderlyingTicker, SecurityType.Equity, Market.USA);
-        public readonly Symbol OptionSymbol = QuantConnect.Symbol.Create(UnderlyingTicker, SecurityType.Option, Market.USA);
-
         public override void Initialize()
         {
             // this test opens position in the first day of trading, lives through stock split (7 for 1), and closes adjusted position on the second day
             SetStartDate(2015, 12, 24);
             SetEndDate(2015, 12, 24);
             SetCash(1000000);
 
-            var equity = AddEquity(UnderlyingTicker);
-            var option = AddOption(UnderlyingTicker);
+            var option = AddOption("GOOG");
 
-            equity.SetDataNormalizationMode(DataNormalizationMode.Raw);
             option.PriceModel = OptionPriceModels.CrankNicolsonFD();
-            option.SetFilter(-2, +2, TimeSpan.FromDays(00), TimeSpan.FromDays(180));
+            option.SetFilter(-2, +2, TimeSpan.Zero, TimeSpan.FromDays(180));
```

Comment:
Cool, maybe we should have an overload which assumes `TimeSpan.Zero` for the earliest expiry? No need to change here, just thinking out loud.

Reply:
Good point. I think that most of people only changes the latest expiry.